apps/agentstack-server/src/agentstack_server/bootstrap.py

```diff
@@ -27,7 +27,7 @@
 
 
 def setup_database_engine(config: Configuration) -> AsyncEngine:
-    return config.persistence.create_async_engine(isolation_level="READ COMMITTED")
+    return config.persistence.create_async_engine(isolation_level="READ COMMITTED", hide_parameters=True)
```

Comment:
Engine was logging secrets, let me know if you want to make this configurable @jezekra1